qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ReportSteps.java

```diff
@@ -80,11 +80,11 @@ public void userEntersEndDate() {
 	@And("user clicks on request report button")
 	public void userclicksOnRequestReport() {
 		reportHistoryPage = runReportsPage.clickOnRequestReport();
-		reportHistoryPage.waitForPageToLoad();
```

Comment:
Whats the difference here. ?

Reply:
@sherrif10 still was failing though passing locally

Reply:
That was how it was behaving initially.

Reply:
ok @sherrif10

Reply:
cc @sherrif10